frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverviewAlerts.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Alert } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { Alert as AlertType } from '@console/internal/components/monitoring/types';
+import { labelsToParams } from '@console/internal/components/monitoring/utils';
+import { sortMonitoringAlerts, getAlertType } from './monitoring-overview-alerts-utils';
+import './MonitoringOverviewAlerts.scss';
+
+interface MonitoringOverviewAlertsProps {
+  alerts: AlertType[];
+}
+
+const MonitoringOverviewAlerts: React.FC<MonitoringOverviewAlertsProps> = ({ alerts }) => {
+  const sortedAlerts = sortMonitoringAlerts(alerts);
+
+  return (
+    <div className="odc-monitoring-alerts">
```

Comment:
We have the same className in the Alerts page. So, CSS properties get applied to here when I visit from the Alerts page.
```suggestion
    <div className="odc-monitoring-overview-alerts">
```